Palace.xcodeproj/project.pbxproj

```diff
@@ -4598,7 +4620,6 @@
 				);
 				MARKETING_VERSION = 1.0.26;
 				PRODUCT_BUNDLE_IDENTIFIER = org.thepalaceproject.palace;
-				PROVISIONING_PROFILE = "2e185b6c-271e-4b02-a05e-860b8c3831f6";
```

Comment:
I'm not sure what these values are (there's another one below), but they were conflicting with the project configuration while I was testing push notifications from Firebase.